docs/product/components/post-summary.html

```diff
@@ -1105,7 +1105,7 @@ <h3 class="s-post-summary--content-title">
 </div>
 {% endhighlight %}
         <div class="stacks-preview--example">
-            <div class="d-flex ai-center gs24 fw-wrap">
+            <div class="d-flex ai-center g24 fw-wrap">
```

Comment:
## No action necessary

This renders differently than `develop`.

<details>
<summary>develop</summary>

![image](https://user-images.githubusercontent.com/647177/197643393-d3d6895a-c8d5-4d37-9de4-9d56921b14f4.png)

</details>

<details>
<summary>This PR</summary>

![image](https://user-images.githubusercontent.com/647177/197643346-bfaa6614-4bbb-4d7f-8c2e-761adf56f360.png)

</details>

I haven't figured out why, but I suspect it's another negative margin difference.